[2021-08-04 13:38:57,598] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_A 2021-08-04T13:38:46.408765+00:00 [queued]>
[2021-08-04 13:38:57,630] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_A 2021-08-04T13:38:46.408765+00:00 [queued]>
[2021-08-04 13:38:57,630] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:38:57,630] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-04 13:38:57,630] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:38:57,644] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_A> on 2021-08-04T13:38:46.408765+00:00
[2021-08-04 13:38:57,648] {standard_task_runner.py:52} INFO - Started process 11930 to run task
[2021-08-04 13:38:57,653] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_A', '2021-08-04T13:38:46.408765+00:00', '--job-id', '2038', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp62bi7t57', '--error-file', '/tmp/tmpkbr8x45l']
[2021-08-04 13:38:57,654] {standard_task_runner.py:77} INFO - Job 2038: Subtask training_model_A
[2021-08-04 13:38:57,705] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag.training_model_A 2021-08-04T13:38:46.408765+00:00 [running]> on host 04479ea62028
[2021-08-04 13:38:57,817] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_A
AIRFLOW_CTX_EXECUTION_DATE=2021-08-04T13:38:46.408765+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-04T13:38:46.408765+00:00
[2021-08-04 13:38:57,817] {python.py:151} INFO - Done. Returned value was: 1
[2021-08-04 13:38:57,861] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_A, execution_date=20210804T133846, start_date=20210804T133857, end_date=20210804T133857
[2021-08-04 13:38:57,903] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-04 13:38:57,947] {local_task_job.py:149} INFO - Task exited with return code 0
